jetbrains-core/src/software/aws/toolkits/jetbrains/services/cloudformation/CloudFormationExplorerNodes.kt

```diff
@@ -54,18 +56,37 @@ class CloudFormationServiceNode(project: Project) : AwsExplorerServiceRootNode(p
     }
 }
 
-class CloudFormationStackNode(project: Project, val stackName: String, private val stackStatus: StackStatus, val stackId: String) :
-    AwsExplorerResourceNode<String>(project, CloudFormationClient.SERVICE_NAME, stackName, AwsIcons.Resources.CLOUDFORMATION_STACK),
+class CloudFormationStackNode(
+    project: Project,
+    val stackName: String,
+    private val stackStatus: StackStatus,
+    val stackId: String
+) :
+    AwsExplorerResourceNode<String>(
+        project,
+        CloudFormationClient.SERVICE_NAME,
+        stackName,
+        AwsIcons.Resources.CLOUDFORMATION_STACK
+    ),
     AwsNodeAlwaysExpandable {
     override fun resourceType() = "stack"
 
+    override fun resourceArn() = stackId
```

Comment:
`stackId` has to be there as a class property in `CloudFormationStackNode ` because it is initialized at the time of object construction:
```
val nodes = response.stacks().filterNotNull().asSequence()
            .filter { it.stackStatus() !in DELETING_STACK_STATES }
            .sortedWith(compareBy(String.CASE_INSENSITIVE_ORDER) { it.stackName() })
            .map { CloudFormationStackNode(nodeProject, it.stackName(), it.stackStatus(), it.stackId()) }
            .toList()
```